masterlist.yaml

```diff
@@ -7084,12 +7099,43 @@ plugins:
       # version: 1.58
       - crc: 0xD7423A4E
         util: 'SSEEdit v4.0.0'
+  - name: 'imp_helm_Immersive_Armors.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12405/'
+        name: 'Improved Closedfaced Helmets Patches'
+    req:
+      - 'imp_helm_legend.esp'
+    after:
+      - 'Hothtrooper44_ArmorCompilation.esp'
+    clean:
+      # version: 1
+      - crc: 0x5C4A5C28
+        util: 'SSEEdit v4.0.2'
   - name: 'imp_helm_IDPM.esp'
     url:
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12405/'
         name: 'Improved Closedfaced Helmets Patches'
+    req:
+      - 'imp_helm_legend.esp'
     after:
       - 'imp_helm_USSEP.esp'
+      - 'Improved Dragon Priest Masks.esp'
+    clean:
+      # version: 1
+      - crc: 0x135E09DE
+        util: 'SSEEdit v4.0.2'
+  - name: 'imp_helm_USSEP.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12405/'
+        name: 'Improved Closedfaced Helmets Patches'
+    req:
+      - 'imp_helm_legend.esp'
+    after:
+      - 'Unofficial Skyrim Special Edition Patch.esp'
```

Comment:
The requires for `imp_helm_legend.esp` is redundant, but the `after` for `Unofficial Skyrim Special Edition Patch.esp` could be replaced with a `req`. As USSEP is not listed as a master in the plugins header.

![image](https://user-images.githubusercontent.com/1944639/57931553-a176b780-78b0-11e9-90a3-10dba792170e.png)